ansible/roles/ufw/readme.md

```diff
@@ -0,0 +1,7 @@
+# UFW Module
+
+This module configures a simple firewall rule using [ufw](https://wiki.ubuntu.com/UncomplicatedFirewall).
+
+To enable it just specify a `ufw_private_interface` (e.g. `eth0`) in your environment variables.
+If you do this, inbound traffic on _all interfaces except that one_ will be blocked.
```

Comment:
that's a good idea. will look into figuring something like that out.

a simpler thing we could do is always allow ssh traffic, though that comes with some (though probably many fewer) risks.